### PR TITLE
Add "Release of Azure Cloud Upload feature" entry

### PR DESCRIPTION
**Pull request by eduardoj**

### CODE REVIEW

**Comment by eduardoj** [comment]
[P2: Blog about new azure upload feature](https://trello.com/c/TUr8BZ7D/792-p2-blog-about-new-azure-upload-feature)

File: _posts/development/2018-08-06-cloud-upload-azure.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+title: Release of Azure Cloud Upload feature
+category: development
+author: "OBS frontend team"
+---
+
+We are proud to announce that the new Azure cloud upload feature has just been released.
+
+You may be familiar with the previous [EC2 Cloud Upload](https://openbuildservice.org/2018/03/01/cloud-upload/) feature.
+This time it is the turn of Azure images.
+The new Azure cloud upload feature allows you to upload Azure images created in OBS to [Microsoft Azure](https://azure.microsoft.com/).
```
Comment (hennevogel) [inline _posts/development/2018-08-06-cloud-upload-azure.md]: Azue Azure Azure. I get it. Can you rephrase this? ;-)

**Comment by hennevogel** [comment]
Deploy preview for *openbuildservice* ready!

Built with commit 4359d4217293676fea8f4f85462546cf0d936cf2

https://deploy-preview-184--openbuildservice.netlify.com

**Comment by eduardoj** [comment]
@hennevogel, @dmarcoux, your suggestions were applied. Please re-review.

File: _posts/development/2018-08-06-cloud-upload-azure.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+In the package overview of an image (your own or another you find in OBS), you can check if your image was successfully built:
+
+<img src="/images/posts/cloud_upload_azure_package_view.png" alt="Azure Cloud Upload Package View" style="max-width: 800px;">
+
+You can click the `images` link in the package view, which will take you to the `binaries` page:
+
+<img src="/images/posts/cloud_upload_azure_binary_page.png" alt="Binaries page" style="max-width: 800px;">
+
+On the `binaries` page you find all binaries and files that were created during a build for a certain repository and architecture.
```
Comment (dmarcoux) [inline _posts/development/2018-08-06-cloud-upload-azure.md]: The `,` is still missing between `page` and `you find`

Comment (eduardoj) [reply _posts/development/2018-08-06-cloud-upload-azure.md]: `,` added.
